### DISCUSSION TITLE
Help: problem with entering event files

### DISCUSSION THREAD

**Discussion by dastout** [HALFpipe/HALFpipe#352] [General] [answered]
Hello,

We are having a problem entering event files. We are using a 3-column format (FSL 3 column). When we attempt to load the files, we are encountering an issue with them being recognized appropriately. Our event file has this naming format, for example:  sub-19_task-acquisition_eCSp.txt. We have 4 other event files as well with the following suffices '_eCSm.txt',  '_cCSp.txt', '_cCSm.txt', '_allpuff.txt'

But we can't enter them into HALFpipe using this structure. It seems to find them if we omit the 'sub-19_' part of the file name, and we are able to progress to the next processing step. However, we subsequently get an error message as it is trying to load the files: [WARNING } Skipping unsupported file because {'suffix': [Unsupported value: eCSp]}. We get this warning for all timing files with extensions listed above. The file path listed for the event file is related to a separate problem that I just posted about in a different thread regarding HALFpipe adding the /ext/ to the front of our working directory.

Is the problem due to how we are naming the event files, the /ext/ prefix to the directory, or something else?

Thank you,

Danny

**Comment by HippocampusGirl** [comment, marked as answer]
Hi Danny,

If you're using BIDS, then the files need to have the `events` suffix. However, it may be easier to manually specify the paths. Could you try if `my_dataset/sub-{subject}/func/task-{task}_{condition}.txt` works?
